### ISSUE TITLE
copySelection exception when the selection contains links

### ISSUE THREAD

**Issue by aspirio187** [NorthwoodsSoftware/GoJS#210]
Hi,

After using `e.diagram.commandHandler.selectAll()`, which will select links and nodes, if you try to use the function `e.diagram.commandHandler.copySelection()`. The app will crash and display this error:

```bash
Cannot read properties of null (reading 'iteratorValues')
TypeError: Cannot read properties of null (reading 'iteratorValues')
    at CommandHandler.copyToClipboard (http://localhost:3000/static/js/bundle.js:56218:139)
    at CommandHandler.copySelection (http://localhost:3000/static/js/bundle.js:56193:14)
    at click (http://localhost:3000/static/js/bundle.js:482:36)
    at ContextMenuTool.JD (http://localhost:3000/static/js/bundle.js:47674:28)
    at ContextMenuTool.standardMouseClick (http://localhost:3000/static/js/bundle.js:47627:39)
    at ContextMenuTool.doContextClick (http://localhost:3000/static/js/bundle.js:50687:135)
    at ContextMenuTool.doMouseUp (http://localhost:3000/static/js/bundle.js:50679:12)
    at Diagram.doMouseUp (http://localhost:3000/static/js/bundle.js:52724:24)
    at i.ha (http://localhost:3000/static/js/bundle.js:52658:319)
```

I tried to only select nodes and copy, it does work. It is however, possible to select all and remove.

I am using GoJS with ReactJs.

The parameters `allowCopy`, `allowDelete` and `allowLink` are all set to `true` during diagram initialization.

```javascript
// set your license key here before creating the diagram: go.Diagram.licenseKey = "...";
const diagram = new go.Diagram({
  allowCopy: true,
  allowDelete: true,
  allowLink: true,
  initialAutoScale: go.AutoScale.UniformToFill,
  ...
})
```

**Comment by simonsarris** [comment]
Thanks for reporting this. What version of GoJS are you using?

**Comment by simonsarris** [comment]
We've been able to reproduce this. A fix will be out with the next version, released probably some time in the coming week.

**Comment by aspirio187** [comment]
> Collaborator

I am using the following npm packages:

```json
"gojs": "^3.0.1",
"gojs-react": "^1.1.2",
```

**Comment by simonsarris** [comment]
This has been fixed in 3.0.2, which we just released. Thank you for reporting it.
